pkg/api/pod/util_test.go

```diff
@@ -3365,8 +3317,6 @@ func TestDropPodLifecycleSleepAction(t *testing.T) {
 
 	for i, tc := range testCases {
 		t.Run(fmt.Sprintf("test_%d", i), func(t *testing.T) {
-			featuregatetesting.SetFeatureGateDuringTest(t, utilfeature.DefaultFeatureGate, features.PodLifecycleSleepAction, tc.gateEnabled)
```

Comment:
This seems to be used by the apiserver to drop disabled fields. Can we keep the disablement test for compatibility version? Have a set of instructions here on adding a line to preserve the disablement test: https://github.com/kubernetes/community/blob/59fb122a7eac1077edaa1b911edeb6ada71d699c/contributors/devel/sig-architecture/feature-gates.md#disablement-tests.

kubelet and e2e tests are fine, I think it's just this unit test and `registry/core/pod`

Reply:
Updated, can you take a look again?